docs/network-service-wiring.md

```diff
@@ -0,0 +1,72 @@
+This document aims at identifying the workflow a network service provider (NSP) would use to manage network services (NS) with some network service wirings (NSW).
+
+The use case is a NSP who wants to provide a gold-network NS, compose by a vpngateway at the begining, and then by a firewall and a vpngateway.
+
+To provide the NS, the NSP will first have to create the NS with the following k8s object creation:
```

Comment:
Some section breaks might be helpful to the reader.  For example, you could consider a structure like:

- The Problem the SP is trying to solve
- Step the SP takes to solve it in an NSM world
       - Deploy a Network Service
       - Deploy Network Service Wirings
       - Deploy NSEs
       - Deploy consuming Pods

(not necessarily exactly that, but that kind of thing :) ).

Reply:
ok, but I'd like to be sure about what exact steps the SP will follow, since it will probably have side-effect, as mentioned below, and impact on implementation details (datamodels and NSM components communications at least).

What I try to highlight in this doc is how the NSP can __dynamically__ manage the traffic flow inside the NS.